plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -180,9 +181,9 @@ func (r *NodeAuthorizer) authorizeGet(nodeName string, startingType vertexType,
 	return r.authorize(nodeName, startingType, attrs)
 }
 
-// authorizeReadNamespacedObject authorizes "get", "list" and "watch" requests to single objects of a
```

Comment:
If we don't need the list permission, should I create a new function or re-use this function? @liggitt